libs/datastructures/include/hpx/datastructures/tuple.hpp

```diff
@@ -149,12 +149,12 @@ namespace hpx { namespace util {
         // tuple(const tuple& u) = default;
         // Initializes each element of *this with the corresponding element
         // of u.
-        constexpr HPX_HOST_DEVICE tuple(tuple const& /*other*/) = default;
+        constexpr tuple(tuple const& /*other*/) = default;
```

Comment:
Just out of curiosity: is it that this is not needed because of the `= default;`?

Reply:
Yep, exactly. It will automatically make default constructors `__host__` and `__device__`.